views/base.njk

```diff
@@ -65,7 +65,7 @@
 
                 {% if not isAuthenticated %}
                   <li>
-                    <a href="/content/examples/" rel="noopener noreferrer">Example sites</a>
+                    <a href="/content/examples/" rel="noopener noreferrer">Examples</a>
```

Comment:
I'm not sure the noun is otherwise clear here....

Reply:
@tram what do you think?  Portfolio perhaps or something else?  "Example sites" also seems to almost make it sound like they're not real.